.github/workflows/release_icm_relayer.yml

```diff
@@ -6,7 +6,7 @@ name: Release icm-relayer
 on:
   push:
     tags:
-      - "icm-relayer/*"
+      - "icm-relayer-*"
```

Comment:
To confirm, we'll tag releases as `icm-relayer-v1.0.0` and `signature-aggregator-v1.0.0` moving forward?